balancer/frontend-v2
# Hotfix/update voting gauges

# Description

Ran `npm run generate:voting-gauges` to update and add newly added gauges. 

## Type of change


## How should this be tested?

- Ensure gauge list looks correct
- Ensure new gauges show. 

## Checklist:
